File: app/src/main/java/com/zerofate/template/ZTApplication.java
package com.zerofate.template;

import android.app.Application;
import android.util.Log;

import com.blankj.utilcode.util.LogUtils;
import com.facebook.stetho.Stetho;
import com.squareup.leakcanary.LeakCanary;
import com.zerofate.androidsdk.util.Utils;

/**
 * Created by Administrator on 2017/10/23.
 */

public class ZTApplication extends Application {
    private static final String TAG = "ZTApplication";

    @Override
    public void onCreate() {
        super.onCreate();
        LeakCanary.install(this);
        LogUtils.getConfig().setGlobalTag("ZeroTemplate");
        Log.d(TAG,
                "onCreate: app module BuildConfig.DEBUG == " + BuildConfig.DEBUG + " & sdk module BuildConfig.DEBUG == "
                        + Utils.isDebug());
        Stetho.initializeWithDefaults(this);
    }
}

File: app/src/main/java/com/zerofate/template/ZTApplication.kt
package com.zerofate.template

import android.app.Application
import android.util.Log

import com.blankj.utilcode.util.LogUtils
import com.facebook.stetho.Stetho
import com.squareup.leakcanary.LeakCanary
import com.zerofate.androidsdk.util.Utils

import timber.log.Timber

/**
 * Created by Administrator on 2017/10/23.
 */

class ZTApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        LeakCanary.install(this)
        LogUtils.getConfig().setGlobalTag("ZeroTemplate")
        Stetho.initializeWithDefaults(this)
        Timber.plant(Timber.DebugTree())
    }

    companion object {
        private val TAG = "ZTApplication"
    }
}
